dev-libs/libunibreak/libunibreak-4.0.recipe

```diff
@@ -0,0 +1,81 @@
+SUMMARY="Line and word breaking library"
+DESCRIPTION="An implementation of the line breaking and word breaking \
+algorithms as described in Unicode Standard Annex 14 and Unicode Standard \
+Annex 29."
+HOMEPAGE="http://vimgadgets.sourceforge.net/libunibreak/"
+COPYRIGHT="2016 Andreas Röver"
+LICENSE="Zlib"
+REVISION="1"
+SOURCE_URI="https://github.com/adah1972/libunibreak/archive/libunibreak_4_0.tar.gz"
+CHECKSUM_SHA256="8ea270d2d9dea6b633f7de1a4a624163c3549b4b6f5031f08c6b96d704d58aa4"
+SOURCE_DIR="libunibreak-libunibreak_4_0"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion=3.1.0
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	libunibreak$secondaryArchSuffix = $portVersion
+	lib:libunibreak$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku${secondaryArchSuffix}
+	"
+
+PROVIDES_devel="
+	libunibreak${secondaryArchSuffix}_devel = $portVersion
+	devel:libunibreak$secondaryArchSuffix = $libVersionCompat
+	devel:liblinebreak$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	libunibreak$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:automake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage libunibreak$secondaryArchSuffix \
+	"$libDir"/libunibreak.so.$libVersion
+
+BUILD()
+{
+	./autogen.sh
+	runConfigure ./configure
```

Comment:
```diff
-	./autogen.sh
-	runConfigure ./configure
+	runConfigure ./autogen.sh
```

This is because `autogen.sh` [runs configure automatically](https://github.com/adah1972/libunibreak/blob/23aa7415254420de0aa4186e72f602257762c9ec/autogen.sh#L12)